Bio/File.py

```diff
@@ -171,10 +177,12 @@ def read(self, size=-1):
         return saved + self._handle.read(size)
 
     def saveline(self, line):
+        """Save the line of the File in memory."""
         if line:
             self._saved = [line] + self._saved
 
     def peekline(self):
+        """Look at the first line saved in memory."""
```

Comment:
How about:

```python
"""Return the next line in the file, but do not move forward though the file."""
```

Again, does that make sense to you as someone who is not (yet) familiar with the ``UndoHandle``class?